src/services/uniswaps.execute.tsx

```diff
@@ -55,8 +46,9 @@ export class UniswapExecutionManager {
       if (swapProcess.txHash) {
         tx = await signer.provider.getTransaction(swapProcess.txHash)
       } else {
-        const call = await uniswap.getSwapCall(action, estimate, srcAddress, destAddress)
-        tx = await signer.sendTransaction(call)
+        const personalizedStep = await personalizeStep(signer, step)
+        const { tx: transaction } = await Lifi.getStepTransaction(personalizedStep)
+        tx = await signer.sendTransaction(transaction)
```

Comment:
We should have the same handling as in 1inch here: update the status in between `getStepTransaction` and `sendTransaction`.